core/debugger/remote_debugger.cpp

```diff
@@ -38,6 +38,7 @@
 #include "core/input/input.h"
 #include "core/object/script_language.h"
 #include "core/os/os.h"
+#include "servers/display_server.h"
```

Comment:
`core` shouldn't depend on `servers`, so you'll have to figure out a different way.

Reply:
OK. I'll add a callback mechanism or something like that.

Reply:
Done!